frontend/hub/access/roles/Roles.tsx

```diff
@@ -29,7 +28,7 @@ export function HubRolesTable() {
 
   const view = usePulpView<Role>({
     url: pulpAPI`/roles/`,
-    keyFn: nameKeyFn,
+    keyFn: (role: { pulp_href: string }) => role.pulp_href,
```

Comment:
This line might cause issues as the keyFn needs to not change between rerenders.